test/system/200-pod.bats

```diff
@@ -478,26 +483,36 @@ spec:
         skip "only meaningful for rootful"
     fi
 
+    mkdir /dev/foo
+    mknod /dev/foo/null b 1 3
     local name1="resources1"
```

Comment:
There really isn't a need for all the numberized variables. Please use "cname" or "containername", without creating unique variables for each one. Same applies to all the other numberized variables.